lib/routes/index.js

```diff
@@ -9,11 +9,12 @@ module.exports = function (app, options, next) {
     require('./healthcheck')(app);
     require('./assets')(app);
     require('./builds')(app);
-    require('./checks')(app);
     require('./packages')(app);
     require('./release-line')(app);
     require('./promote')(app);
     require('../npm/routes')(app);
+    require('./swagger')(app);
```

Comment:
should swagger go first since the definitions are used elsewhere? or I guess it doesn't matter in terms of how the parser works?

Reply:
I don't think it actually matters, but for clarity I'll move it after the healthcheck